### ISSUE TITLE
Feature Request: Ipv6 enabled by default

### ISSUE THREAD

**Issue by sameersbn** [bitnami/bitnami-docker-mongodb#97]
**TL;DR**

Enable IPv6 with the following configuration

```config
net:
  bindIpAll: true
  ipv6: true
```

**Description**

Kubeapps (a kubernetes dashboard with helm and kubeless integration) uses the bitnami/mongodb image and recently we faced an issue where a user had a ipv6 only kubernetes cluster and he could not get the mongodb connection in kubeapps going because mongod wasn't listening on ipv6 addresses (ref: https://github.com/kubeapps/kubeapps/issues/340). This can be classified as a usage issue where kubeapps is expected to override the mongod configuration as per the application requirements. Since our investigation revealed that enabling ipv6 by default would not have any functional impact on ipv4 only environments, we think it makes sense to enable ipv6 by default in the mongodb container image configuration.

IPv6 in `mongod` needs to be enabled by adding the `--ipv6` flag on the command line or adding `net.ipv6: true` in the config. We can enable ipv6 by default in the config and by additionally adding `net.bindIpAll: true` (and removing `net.bindIp: 0.0.0.0`)  we can ensure that the mongod will bind on all available addresses and continue to operate normally in ipv4 only environments.

**Comment by sameersbn** [comment]
ipv6 support seems to be enabled by default since `bitnami/mongodb:3.6.5-r31`. thanks for updating this so quickly :heart:
